client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -15,11 +15,31 @@ varying vec3 tsEyeVec;
 varying vec3 lightVec;
 varying vec3 tsLightVec;
 
-bool normalTexturePresent = false; 
+bool normalTexturePresent = false;
+bool texTileableHorizontal = false;
+bool texTileableVertical = false;
+bool texSeamless = false;
 
 const float e = 2.718281828459;
 const float BS = 10.0;
- 
+
+void getTextureFlags ()
```

Comment:
What's with the extra space here???
Also, this is not part of a class, why do you use the camelCase naming style instead of the lowercase_underscore_style like the rest of the functions in this file use?